test_tools/sigma_verify_rules.py

```diff
@@ -76,8 +115,24 @@ def run_verifier(rules_path, config_file_path):
                     continue
 
                 rule_file_path = os.path.join(dirpath, rule_filename)
-                parsed_rule = sigma_util.get_sigma_rule(
-                    rule_file_path, sigma_config)
+
+                block_because_csv = False
+                for item in ignore['path']:
+                    if item in rule_file_path:
+                        return_rules_with_problems.append(rule_file_path)
+                        block_because_csv = True
+                        logging.info('{0:s} Ignoring because {1:s}'.format(
+                            rule_file_path,item))
+
+                if block_because_csv:
+                    continue
+
+                try:
+                    parsed_rule = sigma_util.get_sigma_rule(
+                        rule_file_path, sigma_config)
+                except Exception:# pylint: disable=broad-except
```

Comment:
why the broad exception here? I really really dislike broad exceptions, and if it's absolutely necessary then please document why that is ht ecase? I have a feeling that there is a finite amount of exceptions that are raised and it woudl be better to handle those rather than have this broad catch all here

Reply:
I will make a comment, I am afraid as this method in the file is made to catch which rule is causing issues, I really do not know what exceptions could be caused, and the idea of that method is catch those and make a useful list for people, so raising the exception and ending the execution would be against the purpose of this method / tool but I will add more comment to it